src/UICrafter.Client/UICrafter.Client.csproj

```diff
@@ -10,8 +10,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.Authorization" Version="8.0.10" />
     <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="8.0.10" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.AzureAppConfiguration" Version="8.0.0" />
     <PackageReference Include="MudBlazor" Version="7.9.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
You are not supposed to use newtonsoft. Dotnet now has their own json serializer. Look at it here:

https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to